products.d/sles_160.yaml

```diff
@@ -54,6 +54,15 @@ translations:
       bulutta ve uçta iş açısından kritik iş yüklerini çalıştırır.
 software:
   installation_repositories: []
+  installation_labels:
+    - label: SLES160-x86_64
+      archs: x86_64
+    - label: SLES160-arch64
```

Comment:
I checked the requirements and it looks correct, but that missing `a` is confusing to me.

Reply:
I think it is to be consistent with S4SAP and there it reach limit of 15 chars...good part is that users won't see this